docs/release-notes.txt

```diff
@@ -4,6 +4,115 @@
  Release Notes
 ###############
 
+**************
+ Version 0.17
+**************
+
+Version 0.17.0
+==============
+
+**Release Date:** September 28, 2021
+
+**New Features**
+
+-  Support the `MMDetection <https://github.com/open-mmlab/mmdetection>`__ library in Model Hub to
+   easily train object detection models. MMDetection provides efficient implementations of popular
+   object detection methods like Mask R-CNN, Faster R-CNN, and DETR on par with Detectron2. In
+   addition, cutting-edge approaches from academia are regularly added to the library.
+
+-  Add a new ``--auto-work-dir`` feature to ``det deploy local``. Setting ``--auto-work-dir
+   /some/path`` will have two effects: first, ``/some/path`` will be bind-mounted into the container
+   (still as ``/some/path``); second, interactive jobs (notebooks, shells, and commands) will run in
+   the provided working directory by default. Note that containers run as the root user by default,
+   so you may want to configure your user with ``det user`` such that interactive jobs run as your
+   regular user.
+
+-  Add the ability to use customizable master configuration templates in ``det deploy aws|gcp``.
+
+-  WebUI: Allow experiment owners to delete their own experiments.
+
+   .. image:: https://user-images.githubusercontent.com/220971/134048799-cd663a75-cb24-4f44-9a8a-c2ff23222cef.png
+
+-  Support configuring an idle timeout for TensorBoards and notebooks.
+
+-  Images: Add an environment image for CPU-only TensorFlow 2.5 and 2.6.
+
+-  WebUI: Add support for batch delete on the experiment list page.
```

Comment:
non-blocking: Maybe combine this with the other WebUI delete functionality?

Reply:
Done!